.github/actions/compute-sdk-test/main.js

```diff
@@ -176,14 +222,21 @@ const mainAsyncTask = async () => {
     try {
       await viceroy.kill();
     } catch(e) {
-      console.error('Could not kill Viceory. Error Below:');
+      console.error('Could not kill test Viceory instance. Error Below:');
```

Comment:
Totally unimportant nit but, as long as you're touching this line anyway, could you fix the spelling of "Viceroy"? :laughing:

Reply:
Didn't even notice that, will do!